acceptance/indy-cli-batches/AS-03-01-steward.batch

```diff
@@ -1,136 +1,136 @@
-- ledger get-nym did=XkZJxs6Uadv6MQeKGGZdZ6
-# #8 AS Steward (steward1) CREATE IdentityOwner (PASS)
-did use 7qFmEyYCXcmUFVied5Sp3b
-ledger nym did=Xm3b3LtJ3UoL5KeYT7ti7j verkey=~GmTyyvE4eHWeSWuiAtmE19
-- ledger get-nym did=Xm3b3LtJ3UoL5KeYT7ti7j
-# #9.1 AS Steward (steward1) REMOVE self (FAIL)
-did use 7qFmEyYCXcmUFVied5Sp3b
-- ledger nym did=7qFmEyYCXcmUFVied5Sp3b role=
-# #9.2 CHECK Steward IS VALID
-ledger nym did=Jt7aMnw77aoaBMyhXUNjt1 verkey=~DT5pLP1wcvsgAzM78sqiR1 role=ENDORSER
-- ledger get-nym did=Jt7aMnw77aoaBMyhXUNjtt
-# #10 AS Steward (steward1) CREATE NetworkMonitor (PASS)
-did use 7qFmEyYCXcmUFVied5Sp3b
-ledger nym did=7e6ttq44HKVSHitCE3BLXv role=NETWORK_MONITOR verkey=~9NGbkySmcB9d6rZ7pPxwtY
-# #11.1 AS Steward (steward1) REMOVE NetworkMonitor (PASS)
-ledger nym did=7e6ttq44HKVSHitCE3BLXv role=
-# #11.2 CHECK NetworkMonitor IS INVALID
-did use 7e6ttq44HKVSHitCE3BLXv
-- ledger get-validator-info nodes=Node1
-# #12.1 AS Steward (steward1) RESTORE NetworkMonitor (PASS)
-did use 7qFmEyYCXcmUFVied5Sp3b
-ledger nym did=7e6ttq44HKVSHitCE3BLXv role=NETWORK_MONITOR
-# #12.2 CHECK NetworkMonitor IS VALID
-did use 7e6ttq44HKVSHitCE3BLXv
-ledger get-validator-info nodes=Node1
-### Additional cases ###
-did use V4SGRU86Z58d6TV7PBUe6f
-did new seed=RandomAttribOwner000000000000001
-ledger nym did=SvXt2QGwZF1kXTcpd2pJ37 verkey=~TmkMNbfcATrfJopaq4KcdV role=STEWARD
-did new seed=RandomUserForRotateKey0000000001
-ledger nym did=6LKnRH6hWPSpoWu824s5JH verkey=~JyFXPqScFNBSoPG4cgTeb1 role=STEWARD
-# Only owner can edit existing NYMs
-did use SvXt2QGwZF1kXTcpd2pJ37
-did rotate-key seed=RandomUserForRotateKey0NewKey001
-# Not owner can't edit existing NYMs (FAIL)
-did use 6LKnRH6hWPSpoWu824s5JH
-- ledger nym did=SvXt2QGwZF1kXTcpd2pJ37 verkey=~TmkMNbfcATrfJopaq4KcdV
-# Only owners can create ATTRIBs
-did use SvXt2QGwZF1kXTcpd2pJ37
-ledger attrib did=SvXt2QGwZF1kXTcpd2pJ37 raw={"endpoint":{"ha":"127.0.0.1:5555"}}
-# Only owners can edit ATTRIBs
-ledger attrib did=SvXt2QGwZF1kXTcpd2pJ37 raw={"endpoint":{"ha":"127.0.0.1:6666"}}
-# Not owner can't create attrib (FAIL)
-did use SvXt2QGwZF1kXTcpd2pJ37
-- ledger attrib did=6LKnRH6hWPSpoWu824s5JH raw={"endpoint":{"ha":"127.0.0.1:5555"}}
-# Not owner can't edit attrib (FAIL)
-did use 6LKnRH6hWPSpoWu824s5JH
-- ledger attrib did=SvXt2QGwZF1kXTcpd2pJ37 raw={"endpoint":{"ha":"127.0.0.1:5555"}}
-# Steward can create SCHEMA:
-ledger schema name=StewardSchema version=1.0 attr_names=name,age
-# Steward can create CLAIM_DEF (will fail because of wrong schema id):
-- ledger cred-def schema_id=1 signature_type=CL primary={"n":"1","s":"2","rms":"3","r":{"age":"4","name":"5"},"rctxt":"6","z":"7"} tag=1
-# Steward can't RESTART POOL
-did use SvXt2QGwZF1kXTcpd2pJ37
-- ledger pool-restart action=start datetime=2020-01-25T12:49:05.258870+00:00 nodes=Node4
-# Steward can get validator-info
-did use SvXt2QGwZF1kXTcpd2pJ37
-ledger get-validator-info nodes=Node1
+# setup environment
```

Comment:
On a clean clone of the repo: 
```
root@73570c5a4ed8:/indy-node# git diff -w
warning: CRLF will be replaced by LF in acceptance/indy-cli-batches/AS-03-01-identity-owner-anyone-can-write.batch.
The file will have its original line endings in your working directory.
warning: CRLF will be replaced by LF in acceptance/indy-cli-batches/AS-03-01-steward.batch.
The file will have its original line endings in your working directory.
```

Due to 

> # Set the default behavior, in case people don't have core.autocrlf set.
> * text=auto

in .gitattributes.

If LF is the desired line ending, these files have to change either in this PR or another one.